modules/ROOT/pages/api-groups-manage-api-group-instance-contracts.adoc

```diff
@@ -5,6 +5,18 @@ endif::[]
 
 After you grant access to API Group instances in Exchange, a contract is automatically created for it. You can then manage these contracts from API Manager.
 
+The API Instance contract details are displayed on the API instance page at two levels: all contracts that are directly with the API instance (API instance contracts) and all contracts made with the API groups that consists of this API instance.
+
+An application can have more than one contract with the different APIs in an API Group. When you view the API instance contracts at the API Group-level, the contracts are sorted by the applications that are in contract with the API Group. The contract that is currently active within an API Group is displayed as `Applied`.
```

Comment:
I don't think that is clear enough what an `Applied` contract means and why this change was introduced. Also, we should specify how the applied contract is selected. 
I don't understand what are you trying to say with `An application can have more than one contract with the different APIs in an API Group.`

Reply:
Done! Thanks!